src/core_modules/capture-core/components/ListView/types/listView.types.js

```diff
@@ -1,17 +1,19 @@
 // @flow
 
-import { dataElementTypes, OptionSet } from '../../../metaData';
+import { dataElementTypes } from '../../../metaData';
 import type {
     FilterData,
 } from '../../FiltersForTypes';
 
+export type Options = Array<{text: string, value: any}>;
```

Comment:
Can we define one Options type only and use this around. If I am not wrong this is also defined in the `selectBoxes.types.js` file. Perhaps we can reuse one or the other?